### ISSUE TITLE
Switching system themes in GNOME does not update color scheme

### ISSUE THREAD

**Issue by Siilwyn** [charmbracelet/glow#262]
When switching between a light and dark theme on GNOME (Fedora 33), the Glow colour scheme except from the background does not update. This issue has been around since v1.0.0, (love using Glow!) always thought it was a terminal bug however I recently noticed that this bug is not present in another TUI ([dijo](https://github.com/NerdyPepper/dijo)). The colours persist even when navigating between files in Glow.

Screenshots to clarify:

## Light theme
![Screenshot from 2021-03-27 11-00-49](https://user-images.githubusercontent.com/5701149/112717301-014a3a80-8eec-11eb-8683-ad6019bbe3d5.png)

## Dark theme, switched while glow is open
![Screenshot from 2021-03-27 11-01-12](https://user-images.githubusercontent.com/5701149/112717300-00b1a400-8eec-11eb-968e-5f1266a3b542.png)

## Dark theme
![Screenshot from 2021-03-27 11-01-39](https://user-images.githubusercontent.com/5701149/112717299-00190d80-8eec-11eb-9637-9eedb9e3c29c.png)

**Comment by muesli** [comment]
Are you talking about switching the theme while glow is running? The theme is only checked on startup (at least currently).

**Comment by Siilwyn** [comment]
@muesli Yes exactly. It might seem like an edge case however I believe this is going to be more common once more distros start using automatic light/dark theme settings like Elementary OS is doing.

**Comment by muesli** [comment]
Quick update:

While not impossible, this might be really tricky to support. We'd have to periodically query the terminal for its current color settings, which - depending on the terminal you're using - can be a time-intensive query. Mind you, we're talking milliseconds, but it's probably not something you'd want to query once per second. I have to do some more tests and benchmarks with various terminals.
